src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/*****************************************************************************/
+#ifndef HARDWARE_INTRINSIC
+#error Define HARDWARE_INTRINSIC before including this file
+#endif
+/*****************************************************************************/
+
+// clang-format off
+
+#ifdef FEATURE_HW_INTRINSICS
+// ***************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
```

Comment:
I thought `IsSupported` would have entry here, but yes, it's not needed. So, wondering if we should still get an empty file or have it as part of a PR that adds anything relevant in it?

Reply:
The only reason for keeping this here is it gets it done now - I can then write API PRs without being concerned about the dependencies of which one adds the files.

That includes the `hwintrinsiclistarm64.h` and `CMakeLists.txt` changes too.

Reply:
I think its better to get it done "now" as it will help avoid conflicts after